numpyro/infer/util.py

```diff
@@ -157,7 +157,8 @@ def transform_fn(transforms, params, invert=False):
     return {k: transforms[k](v) if k in transforms else v for k, v in params.items()}
 
 
-def constrain_fn(model, model_args, model_kwargs, params, return_deterministic=False):
+def constrain_fn(model, model_args, model_kwargs, params, 
+                 include_param_sites=False, return_deterministic=False):
```

Comment:
I think you can remove those `include_param_sites` flags and keep the True behavior by default.

Reply:
This was for ensuring backward compatibility. I removed these in the last comit

Reply:
I see. I guess it won't affect the old behavior. If users provide param sites, your solution will return expected results, while the current master branch will raise an error - so this is an improvement.